### PR TITLE
OCPP201: SignCertificate and CertificateSigned

### PR DESCRIPTION
**Pull request by Pietfried**
* added handling for security: A02 (SignCertificate, CertificateSigned)

### CODE REVIEW

File: lib/ocpp/v201/charge_point.cpp [hunk 1]
```diff
@@ -1434,6 +1489,98 @@ void ChargePoint::notify_event_req(const std::vector<EventData>& events) {
...
+
+    if (result == ocpp::InstallCertificateResult::Accepted) {
+        response.status = CertificateSignedStatusEnum::Accepted;
+    }
+
+    ocpp::CallResult<CertificateSignedResponse> call_result(response, call.uniqueId);
+    this->send<CertificateSignedResponse>(call_result);
+
+    if (result != ocpp::InstallCertificateResult::Accepted) {
+        this->security_event_notification_req("InvalidChargingStationCertificate", std::optional<CiString<255>>{}, true,
```
Comment (Pietfried) [inline lib/ocpp/v201/charge_point.cpp:1521]:
```suggestion
        this->security_event_notification_req("InvalidChargingStationCertificate", ocpp::conversions::install_certificate_result_to_string(result), true,
```

File: lib/ocpp/v201/charge_point.cpp [hunk 2]
```diff
@@ -1968,10 +2115,15 @@ void ChargePoint::handle_trigger_message(Call<TriggerMessageRequest> call) {
         }
         break;
 
+    case MessageTriggerEnum::SignChargingStationCertificate:
+        EVLOG_info << "got SignChargingStationCertificate";
```
Comment (Pietfried) [inline lib/ocpp/v201/charge_point.cpp:2119]:
```suggestion
```

File: lib/ocpp/v201/charge_point.cpp (hunk 1 above)
Comment (Pietfried) [reply lib/ocpp/v201/charge_point.cpp:1521]: note that tests need to be adjusted when this is applied

**Review by Pietfried — COMMENTED** [review]
Looks like I cant Approve / Request changes because I opened this PR

File: lib/ocpp/v201/charge_point.cpp (hunk 1 above)
Comment (klemmpnx) [reply lib/ocpp/v201/charge_point.cpp]: done-> test update: https://github.com/PionixInternal/ocpp-testing/pull/35

**Review by klemmpnx — APPROVED** [review]
rebased and adapted; pls check again / maybe run updated tests (pr: https://github.com/PionixInternal/ocpp-testing/pull/35) - but for me good to merge!

**Pull request merged**